multimedia/spotify_dl/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                spotify_dl
```

Comment:
can you please use the PyPI name for the port, which appears to be `spotify-dl`? Then you'd use `python.rootname spotify_dl` to change to the underscore, which upstream uses for downloads....

Reply:
Done